demand-capacity-mgmt-backend/src/main/resources/application.yml

```diff
@@ -25,8 +25,8 @@ spring:
     show-sql: true
   datasource:
     url: jdbc:postgresql://${DCM_DATASOURCE_HOST:localhost:5432}/${DCM_DATASOURCE_NAME:dcm}
-    username: ${DCM_DATASOURCE_USER:postgres}
-    password: ${DCM_DATASOURCE_PASS:postgres}
+    username: ${DCM_DATASOURCE_USER:dcm}
+    password: ${DCM_DATASOURCE_PASS:dcm}
```

Comment:
Please do not hard code username and password but make it configurable in a way that you can set/control it in your product helm chart `values.yaml` file.